naurffxiv/src/app/navigation.js

```diff
@@ -0,0 +1,67 @@
+import * as React from 'react';
+import AppBar from '@mui/material/AppBar';
+import Toolbar from '@mui/material/Toolbar';
+import Typography from '@mui/material/Typography';
+import Button from '@mui/material/Button';
+import IconButton from '@mui/material/IconButton';
+import { Container } from "@mui/material";
+import { Grid } from '@mui/material';

```

Comment:
i probably shouldve mentioned this sooner but we should import everything from one source like
```
import { Container, Grid, AppBar, Toolbar, ........ } from '@mui/material';
```
to make it cleaner